File: GotlandFrontend/lib/model/cabin_class_detail.dart
import 'json_field.dart';

class CabinClassDetail {
	final String id;
	final String title;
	final int no;
	final int capacity;
	final int count;

	CabinClassDetail(this.id, this.title, this.no, this.capacity, this.count);

	factory CabinClassDetail.fromMap(Map<String, dynamic> json) =>
		CabinClassDetail(
			json[ID],
			json[TITLE],
			json[NO],
			json[CAPACITY],
			json[COUNT]
		);
}

File: GotlandBackend/Models/CabinClassDetail.cs
using System;

namespace Accidis.Gotland.WebService.Models
{
	public class CabinClassDetail
	{
		public Guid Id { get; set; }
		public string Title { get; set; }
		public int No { get; set; }
		public int Capacity { get; set; }
		public int Count { get; set; }
	}
}
